weed/shell/command_s3_clean_uploads.go

```diff
@@ -82,7 +84,7 @@ func (c *commandS3CleanUploads) cleanupUploads(commandEnv *CommandEnv, writer io
 		fmt.Fprintf(writer, "purge %s\n", deleteUrl)
 
 		err = util.Delete(deleteUrl, "")
-		if err != nil {
+		if err != nil && err.Error() != "" {
```

Comment:
the additional check is necessary?

Reply:
I use jwt token on filler

```
> s3.clean.uploads -timeAgo 2h
purge http://fast-api.s3.svc:9090/buckets/cloud/.uploads/b470e03b-4851-48f5-9935-67649eda2f28?recursive=true&ignoreRecursiveError=true
> fs.ls /buckets/cloud/.uploads/
```
but s3.clean.upload remove files without jwt token